### ISSUE TITLE
check_date key

### ISSUE THREAD

**Issue by matkoniecz** [simonpoole/beautified-JOSM-preset#171]
Vespucci is showing old unmodified objects of fragile types with heavy rotation (shops) as suspect.

It makes sense, and often such objects may be marked as still existing and OK by adding some additional detail - `wheelchair`, `opening_hours`, sometimes also for example `level` or moving it to a more correct location.

But sometimes object is fully correctly mapped, there are no reasonable tags that may be added and it is still not edited for year or more.

In such situation adding `check_date` seems to be the best solution ( https://wiki.openstreetmap.org/wiki/Key:check_date ), but it is not in Vespucci presets at this moment and it would be nice to have way to add it that would autofill the current date.

**Comment by simonpoole** [comment]
I'm not sure why you believe it is not in the default presets, it's been ther for at least a year, if not longet. And as a loadable preset for literally years.

**Comment by matkoniecz** [comment]
I opened Vespucci 14.0.9

I found and selected `shop=bag`, `name=Vip`, `opening_hours 10:00-18:00` node that is not edited for more than one year (`wheelchair` was not a viable addition, as the shop was closed).

I started typing `check_date` and there was no autocompletion of key or its value.

I now tested it again and discovered that one may search "check" in presets what will find "Add or update check date" with JS icon. As bonus, "Add or update check date" will now appear below tags in "properties" pane.

Is it possible to add `check_date` availability in autocomplete on the properties tab? 

Is it possible to have "Add or update check date" below tags in "properties" pane for objects that are outdated?
